### PR TITLE
Make the NPCs refuse to interact after accusation

### LINKED ISSUE #1: Ensure NPCs become unhappy when falsely accused
Modify NPC behaviour so that they become unhappy when falsely accused.

### PR DESCRIPTION
**Pull request by Comprs**
This closes #1.

### CODE REVIEW

File: game/core/src/me/lihq/game/ConversationManagement.java [hunk 1]
```diff
@@ -71,6 +71,17 @@ public void startConversation(NPC npc)
...
+        // conversation early and with the appropriate text.
+        if (this.tempNPC.isAccused()) {
+            speechboxMngr.addSpeechBox(new SpeechBox(
+                "This person is no longer willing to cooperate", 5
+            ));
+            finishConverstation();
+            return;
+        }
+
         speechboxMngr.addSpeechBox(new SpeechBox(this.tempNPC.getName(), this.tempNPC.getSpeech("Introduction"), 5));
```
Comment (jacobsax) [inline game/core/src/me/lihq/game/ConversationManagement.java:86]: Unnecessary space here

File: game/core/src/me/lihq/game/people/NPC.java [hunk 2]
```diff
@@ -37,6 +37,11 @@
     private boolean isVictim = false;
 
     /**
+     * This indicates whether the NPC has been accused.
+     */
+    private boolean hasBeenAccused = false;
```
Comment (jacobsax) [inline game/core/src/me/lihq/game/people/NPC.java:42]: All of these variables values should really be set in an initialiser function

Comment (jacobsax) [reply game/core/src/me/lihq/game/people/NPC.java:42]: Also a multi block comment isn't necessary here, it should really only be one line

**Review by jacobsax — CHANGES_REQUESTED** [review]

File: game/core/src/me/lihq/game/people/NPC.java (hunk 2 above)
Comment (Comprs) [reply game/core/src/me/lihq/game/people/NPC.java:42]: This is a javadoc documentation, not a comment. This is the only available style.

Comment (jacobsax) [reply game/core/src/me/lihq/game/people/NPC.java:42]: You disappoint me

File: game/core/src/me/lihq/game/ConversationManagement.java (hunk 1 above)
Comment (Comprs) [reply game/core/src/me/lihq/game/ConversationManagement.java:86]: If this space is unnecessary, then the one above this block should also be unnecessary. Is this so?

File: game/core/src/me/lihq/game/people/NPC.java (hunk 2 above)
Comment (Comprs) [reply game/core/src/me/lihq/game/people/NPC.java:42]: No, javadoc disappoints you

**Pull request merged**